addons/mail/models/mail_thread.py

```diff
@@ -2896,6 +2896,18 @@ def _message_log_batch(self, bodies, subject=False,
                        for record in self]
         return self.sudo()._message_create(values_list)
 
+    def _post_notification(self, notification_type, data=None):
+        """Post a message notification on this thread.
+
+        :notification_type str: Type of the notification.
+        :data dict: Optional data used to compute the message body.
+        """
+        body = Markup(
```

Comment:
This makes no sense: why logging as a *comment* JSON data ? Are you going to send that to customers by email ? If not, this should not exists.